source/deployment-options/offline-installation.rst

```diff
@@ -225,6 +225,14 @@ Installing the Wazuh manager
 
     .. include:: /_templates/installations/wazuh/common/check_wazuh_manager.rst
 
+#. Save the Wazuh indexer username and password into the Wazuh keystore using the wazuh-keystore tool: 
```

Comment:
Wazuh keystore -> Wazuh manager keystore

Wazuh indexer, Wazuh dashboard, and Filebeat also have their own keystore, we have to name our components properly

Reply:
Done in [Wazuh components are named appropriately](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/ef178ca855e5c05eca6c5c8705a7e54e2199f3bb) commit.